CHANGELOG.md

```diff
@@ -9,6 +11,7 @@
     * Update react-native to version 0.58+ and update App Center SDK packages.
     * Edit **app/build.gradle** to replace `compile` by `implementation`.
     * Run `react-native link` again.
+- Support a 256 character User ID string that applies to crashes, error, and push logs. Settable via `AppCenter.setUserId()`.
```

Comment:
>a 256 character User ID string

This could be misleading. When I read it, I read it as the user string must be 256 characters. We should describe it so that it is clear 256 characters is maximum

Reply:
Fixed it to what @dhei recommended.